ui/src/components/erc/Substitution/Substitution.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import httpRequests from '../../../helpers/httpRequests';
-import config from '../../../helpers/config';
+//import config from '../../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.